package.json

```diff
@@ -10,13 +10,9 @@
     "prestart-production": "gulp build -p",
     "start-production": "node pm2Start",
     "lint": "npm run lint-js && npm run lint-json",
-    "lint-challenges": "jsonlint-cli seed/challenges/**/*.json",
-    "lint-server": "jsonlint-cli server/*.json",
-    "lint-resources": "jsonlint-cli server/resources/*.json",
-    "lint-utils": "jsonlint-cli server/utils/*.json",
     "prelint-js": "npm run create-rev",
     "lint-js": "eslint --ext=.js,.jsx gulpfile.js server/ common/ config/ client/",
-    "lint-json": "npm run lint-server && npm run lint-challenges && npm run lint-resources && npm run lint-utils",
+    "lint-json": "eslint --ext=.json seed/challenges/ server/",
```

Comment:
I see.

We should go with option 2 so that all json files are linted.

The error is troubling as it assumes the previous spec of JSON where JSON must be an object or an array. Current spec states that JSON must be a value. 

After taking a look under the hood, it looks like eslint-plugin-json uses jshint under the hood. It may be that jshint uses an outdated spec to lint json. This maybe a no go for this plugin. Let me see if I can put something together.